### PR TITLE
EB-271: Add litecoin address form field

### PR DESCRIPTION
**Pull request by sszaharovd**
### Description
`Django forms` allows us to validate the input data, which increases security and eliminates the possibility of entering incorrect information. Thus, we eliminate ourselves from the appearance of unexpected errors or the failure of the program.

### Implemented
— Create Litecoin address form field for validation ones.
— Add tests for Litecoin address form field validation logic.
— Add Litecoin constants.
— Write documentation for Litecoin and Bitcoin address form field section.

### References
- Litecoin Trezor P2SH — https://blog.trezor.io/litecoins-new-p2sh-segwit-addresses-843633e3e707
- Litecoin information — https://github.com/litecoin-project/litecoin/issues/312
- What is SegWit Litecoin address — https://github.com/sipa/bech32/blob/master/ref/python/segwit_addr.py
- Litecoin python library — https://chaining.ru/2018/05/31/litecoin-core-v0-16-0-release-litecoin-project/
- Bitcoin BIP-173 — https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki
- Bitcoin Bit — https://github.com/ofek/bit/blob/e5640cbc79c183b8c4051b46b3ccf3e813e021d3/bit/base58.py
- Bitcoin SegWit — https://github.com/sipa/bech32/blob/master/ref/python/segwit_addr.py

### CODE REVIEW

File: crypto_django/forms/fields/addresses/bitcoin.py [hunk 1]
```diff
@@ -6,10 +6,11 @@
...
 
 from crypto_django.constants.address import (
-    BECH32_PREFIX,
-    P2PKH_PREFIX,
-    P2SH_PREFIX,
+    BITCOIN_BECH32_PREFIX,
+    BITCOIN_P2SH_PREFIX,
+    BITCOIN_P2PKH_PREFIX,
     REQUIRED_BITCOIN_ADDRESS_LENGTH,
+    MIN_BITCOIN_ADDRESS_LENGTH,
```
Comment (anastasiia-bilova) [inline crypto_django/forms/fields/addresses/bitcoin.py]: Wrong order.

File: crypto_django/forms/fields/addresses/litecoin.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+from bit.base58 import b58decode_check
+from django import forms
+from django.core.exceptions import ValidationError
+
+from crypto_django.constants.address import (
+    LITECOIN_BECH32_PREFIX,
+    LITECOIN_P2SH_PREFIX,
+    LITECOIN_P2PKH_PREFIX,
+    REQUIRED_LITECOIN_ADDRESS_LENGTH,
+    MIN_LITECOIN_ADDRESS_LENGTH,
```
Comment (anastasiia-bilova) [inline crypto_django/forms/fields/addresses/litecoin.py]: Wrong order.

File: crypto_django/forms/fields/addresses/litecoin.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    }
+
+    def to_python(self, value):
+        """
+        Validate Litecoin address.
+
+        References:
+            - https://blog.trezor.io/litecoins-new-p2sh-segwit-addresses-843633e3e707
+            - https://github.com/litecoin-project/litecoin/issues/312
+            - github.com/sipa/bech32/blob/master/ref/python/segwit_addr.py
```
Comment (anastasiia-bilova) [inline crypto_django/forms/fields/addresses/litecoin.py]: Why without `https`?

File: crypto_django/tests/forms/fields/addresses/test_bitcoin.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        self.not_valid_address = '16DWEqF6pX314CjV5u2dmiMhcXtyY6z7odzz'
+        self.valid_address = '16DWEqF6pX314CjV5u2dmiMhcXtyY6z7od'
+        self.segwit_address = '3MLiqxr3iyER1mZkrdvt83c99P1bsGjqH2'
+
+        self.bitcoin_address_field = BitcoinAddressField()
+
+    def test_required_length_error(self):
+        """
+        Case: validate address with invalid required length (less than needed).
+        Expect: ValidationError is raised.
```
Comment (anastasiia-bilova) [inline crypto_django/tests/forms/fields/addresses/test_bitcoin.py]: `ValidationError is raised` > `validation error message is raised`

File: crypto_django/forms/fields/addresses/bitcoin.py (hunk 1 above)
Comment (sszaharovd) [reply crypto_django/forms/fields/addresses/bitcoin.py]: done

File: crypto_django/forms/fields/addresses/litecoin.py (hunk 2 above)
Comment (sszaharovd) [reply crypto_django/forms/fields/addresses/litecoin.py]: done

File: crypto_django/forms/fields/addresses/litecoin.py (hunk 3 above)
Comment (sszaharovd) [reply crypto_django/forms/fields/addresses/litecoin.py]: done

File: crypto_django/tests/forms/fields/addresses/test_bitcoin.py (hunk 4 above)
Comment (sszaharovd) [reply crypto_django/tests/forms/fields/addresses/test_bitcoin.py]: done

**Pull request merged**